packages/framework/fluid-framework/package.json

```diff
@@ -22,11 +22,24 @@
 				"types": "./dist/index.d.ts",
 				"default": "./dist/index.js"
 			}
+		},
+		"./legacy": {
+			"import": {
+				"types": "./lib/legacy.d.ts",
+				"default": "./lib/legacy.js"
+			},
+			"require": {
+				"types": "./dist/legacy.d.ts",
+				"default": "./dist/legacy.js"
+			}
 		}
 	},
 	"main": "lib/index.js",
 	"types": "lib/index.d.ts",
 	"scripts": {
+		"api": "fluid-build . --task api",
+		"api-extractor:commonjs": "flub generate entrypoints --outFileAlpha legacy --outDir ./dist",
+		"api-extractor:esnext": "flub generate entrypoints --outFileAlpha legacy --outDir ./lib --node10TypeCompat",
```

Comment:
@tylerbutler, @jason-ha - This package doesn't use API tags, but 'flub generate entrypoints' still correctly generates the 'legacy.d.ts' stub which makes attw pass.

Alternatives:
* Could remove the 'api-extractor:commonjs', since it doesn't do the node10TypeCompat.
* Could remove the --outFileAlpha, since we're not using tags.
* Could remove 'api' task all together to push SDK customers to Node10/Bundler

Reply:
> Could remove 'api' task all together to push SDK customers to Node10/Bundler

I assume you meant Node16/Bundler? If so I prefer that option. My hope is that requiring Node16 doesn't preclude many
people from experimenting in the Beta timeframe, and I really don't think Node10 will stay viable for much longer.

Reply:
And actually with Jason's latest changes removing the --outFileAlpha causes an error that there are no exports matching
the entrypoints.

```
Error: There are no package exports matching requested output entrypoints:
        lib/alpha.d.ts
        lib/beta.d.ts
        lib/public.d.ts
```

And outFileAlpha overwrites the typescript generated types, which probably isn't what we want.

I think the best solution if we need node10 compat here is to rename legacy.ts to legacy-src.ts and let the rollup do
its thing for the legacy entrypoint.